samples/netcore/Startup.cs

```diff
@@ -86,8 +92,15 @@
                 Console.WriteLine($"    Config with key {item.Key} has value {item.Value}");
             }
             Console.WriteLine("End - printing config settings");
-
         }        
+
+        private static async Task TerminateAfterSeconds(int seconds)
+        {
+            int timeout = new Random().Next(-60,60) + seconds;
+            Utils.LogMessage($"Terminating after {timeout} seconds");
+            await Task.Delay(TimeSpan.FromSeconds(timeout));
+            Environment.Exit(0);

```

Comment:
if someone puts in something bad for TERMINATE_AFTER_SECONDS here (like something other than 60, say 30) this will do a weird thing where it'll send a negative timespan to task.delay....which is probably not the ideal thing to have happen. 

instead of this, I'd recommend using the TERMINATE-AFTER-SECONDS be something like the mean value and pick a random value between 0 and 2x terminate. That way it will never go negative.

Reply:
Good call, updated. Did the minimum to be 20 seconds.